<launch>
    <!-- openzen IMU  Sensor-->
    <node pkg="openzen_sensor" type="openzen_sensor_node" name="imu_node" output="screen">
        <param name="baudrate" value="115200" />
        <param name="frame_id" value="imu_link" />
    </node>
        
    <!-- Publish static transform from base_footprint to imu_link -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="base_link_to_imu_link" args="0 0 0.05 1.5707963 0 0  /base_link /imu_link"/>
</launch>
